docs/content/docs/upgrade_guide.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Upgrade Guide"
```

Comment:
Just a note for the future: you can use `git mv` which preserves the history when moving files

Reply:
I did do that but I wonder if it didn't like it with how many other changes I made.